router/router.go

```diff
@@ -0,0 +1,32 @@
+package router
+
+import (
+	"budget-helper/users"
+	"net/http"
+)
+
+type Router struct {
+	Handler  *http.ServeMux
+	UserRepo *users.UserRepo
+}
+
+func NewRouter(u *users.UserRepo) *Router {
+	h := http.NewServeMux()
+
+	return &Router{
+		Handler:  h,
+		UserRepo: u,
+	}
+}
+
+func (rt *Router) RegisterRoutes() {
+	// -- REGISTER ALL ROUTES HERE --
+	rt.Handler.HandleFunc("/", rt.handleHome)
+	rt.Handler.HandleFunc("/users", rt.handleUsers)
+}
+
+//func wrapWithDB(db *database.Database, f func(*database.Database, http.ResponseWriter, *http.Request)) func(http.ResponseWriter, *http.Request) {
```

Comment:
We have Git if we ever want to find old code. Do not leave dead code in project.

I once worked with a customer where 2/3 of their _10 gigabytes_ of code was commented old dead code. Never have I ever wanted to `rm -rf` a project more than that time.